### DISCUSSION TITLE
assets

### DISCUSSION THREAD

**Discussion by gianlore** [Keriew/augustus#432] [General]
Dove trove il file "assets" me lo chiede alla partenza.
Forse è per quello che nel gioco ci sono moltissimi malfunzionamenti.

**Comment by crudelios** [comment]
English please 🙂

Try: https://augustus.josecadete.net/download/latest/development/windows

**Reply by gianlore** [reply]
123 / 5000
Risultati della traduzione
And just what I felt. He asks me for the "assets" directory. The game starts the same, but many things don't work.

**Comment by gianlore** [comment]
there is no "assets" directory in the zip file
